src/Maker/MakeTwigExtension.php

```diff
@@ -59,12 +66,29 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             AbstractExtension::class,
             TwigFilter::class,
             TwigFunction::class,
+            'App\\Twig\\Runtime\\'.$name.'Runtime'
         ]);
 
         $generator->generateClass(
             $extensionClassNameDetails->getFullName(),
             'twig/Extension.tpl.php',
-            ['use_statements' => $useStatements]
+            ['use_statements' => $useStatements, 'runtime_class_name' => $name.'Runtime::class']
+        );
+
+        $runtimeClassNameDetails = $generator->createClassNameDetails(
+            $name,
+            'Twig\\Runtime\\',
+            'Runtime'
+        );
```

Comment:
Can we create this up above line 69 so that we can use it on 69 - e.g. `$runtimeClassNameDetails->getFullName()`

Reply:
I've read more of the internal of the generator and indeed it appear it is more clever to do that instead of "hacky" string concat/replace

I ve rewritten as you point to